source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
-                           <specDesc key="att.bifoliumSurfaces" atts="inner.recto"/>
-                           <specDesc key="att.bifoliumSurfaces" atts="outer.verso"/>
+                           <specDesc key="titlePage"/>
+                           <specDesc key="p"/>
+                           <specDesc key="table"/>
+                           <specDesc key="list"/>
+                           <specDesc key="quote"/>
+                           <specDesc key="lg"/>
                         </specList>
                      </p>
-                     <p>With those attributes, page numbers can be derived from <gi scheme="MEI">foliaDesc</gi>, alongside the information where the content on a given <gi scheme="MEI">surface</gi> is placed on a (bi)folium. Coming back to the example above this might look like so:</p>
+                     <p>The following example shows the encoding of the title page of Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi>. Note the use of the <gi scheme="MEI">lb</gi> element to mark the line breaks present in the original.</p>
                      <p>
                         <figure>
                            <head/>
-                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve" valid="feasible"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/foliadesc-02.txt" parse="text"/></egXML>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/text/text-sample371.txt" parse="text"/></egXML>
                         </figure>
                      </p>
+                     <p>The physical rendition of title page information is often of considerable importance. One approach to this requirement would be to use the <gi scheme="MEI">rend</gi> element, described in chapter <ptr target="#sharedTextRendition"/> to specify the rendition of each of the components of the title page. Another would be to employ a CSS stylesheet. Finally, a module customized for the description of typographic entities such as pages, lines, rules, etc., bearing special-purpose attributes to describe line-height, leading, degree of kerning, font, etc. could be employed.</p>
                   </div>
-                  <div xml:id="foliumdimensions" type="div4">
-                     <head>Specifying page dimensions</head>
-                     <p>Within <gi scheme="MEI">surface</gi>, each <gi scheme="MEI">graphic</gi> element may specify its dimensions using the following attributes:</p>
+                  <div xml:id="physicalProperties" type="div4">
+                     <head>Physical Properties</head>
+                     <p>Even if the section <ptr target="#msdesc"/> is about how do a physical description this part is explaining elements on the medium level.</p>
```

Comment:
This sentence does not make a lot of sense on its own... ?

Reply:
I liked to express that this section is about the physical description of a manifestation. Maybe this is enough to introduce this section. I agree that the proposed sentence is not helpful! The intention was also to have a distinction between encoding sources and encoding physical properties of a source to make sure that it is not the same. 
@ahankinson could you help me to rephrase it?